docker-files-for-Gitlab-CI-rust/parity-eth-linux-ci/Dockerfile

```diff
@@ -1,18 +1,27 @@
-FROM rust:stretch
-LABEL maintainer="Parity Technologies <devops-team@parity.io>"
-LABEL description="This is for CI build stage for Parity Ethereum linux binary."
+# triggers image autoupdate with the new Rust release
+FROM rust:slim as notifier
```

Comment:
that's pretty cool :+1:

Reply:
and weighs not that much, but I consider it a crutch though xD